Vizzuality/widget-editor
# fix/area-intersection-patch

This pr fixes an issue when updating area intersection properties the editor does not catch that this change happened. 

## Testing instructions

I updated the playground with a new field "area intersection id" that will patch the current area intersection property in the editor. This is so we can have the same behavior as people integrating the editor and passing this id down to the editor. 

Play around with pasting these IDS and make sure area intersection behaves correctly. 

1. `01dec423bd8a15a0ad8756f72d1f0788` (Argentina should appear)
2. `35a6d982388ee5c4e141c2bceac3fb72 ` (Spain should appear) 
3. `1971a5e1c491a12e4c8e901fb23fe6c0` (Vietnam should appear) 

Also, try to remove the intersection ID from the input box, and the editor will remove the selection from the UI if a previous intersection is present.

## Pivotal Tracker

[here](https://www.pivotaltracker.com/story/show/175739002)
